src/main/java/com/kwezal/bearinmind/core/course/controller/CourseUserDataController.java

```diff
@@ -27,4 +28,26 @@ public class CourseUserDataController {
     public void enrollUserInCourse(@PathVariable @Min(1) Long courseId) {
         courseUserDataService.enrollUserInCourse(courseId);
     }
+
+    /**
+     * Finds a role in a given course for the logged-in user.
+     *
+     * @param id course ID
+     * @return course role
+     */
+    @GetMapping("/{id}/role")
+    public CourseRole findCourseRoleByCourseId(@PathVariable @Min(1) Long id) {
```

Comment:
Could you add "@Min(1) " annotations to all of the endpoints where id is given?

Reply:
> Could you add "@min(1) " annotation to all of the endpoints where id is given?

Sure, good idea.